src/debby/sqlite.nim

```diff
@@ -117,7 +117,7 @@ proc prepareQuery(
   for i, arg in args:
     if arg.len == 0:
       continue
-    if sqlite3_bind_text(result, int32(i + 1), arg[0].unsafeAddr, arg.len.int32, nil) != SQLITE_OK:
+    if sqlite3_bind_text(result, int32(i + 1), cast[cstring](arg[0].unsafeAddr), arg.len.int32, nil) != SQLITE_OK:
```

Comment:
Can you explain why this cast is needed?

Reply:
The compiler outputs `Warning: unsafe conversion to 'cstring' from 'ptr char'; this will become a compile time error in the future [PtrToCstringConv]` without this, so I thought casting it as a `cstring` was a better idea here

Reply:
In my code I just did arg.cstring and it worked better. Thanks!